data/ke2022daily/ke2022daily-extraction.md

```diff
@@ -0,0 +1,100 @@
+# Extraction for Ke et al. (2022)
+
+[Ke et al. (2022)](https://www.nature.com/articles/s41564-022-01105-z) studied the dynamics of infectious virus and viral RNA shedding for SARS-CoV-2 during acute infection through daily longitudinal sampling of 60 individuals for up to 14 days. Nasal swab and saliva samples were collected daily and tested for University of Illinois at Urbana-Champaign faculty, staff, and students (during the fall of 2020 and spring of 2021) who reported a negative RT–qPCR test result in the past 7 days and were either (1) within 24 h of a positive RT–qPCR result or (2) within 5 days of exposure to someone with a confirmed positive RT–qPCR result. Nasal swab samples were also cultured but the results were not included in our dataset. The raw data was obtained from [here](https://static-content.springer.com/esm/art%3A10.1038%2Fs41564-022-01105-z/MediaObjects/41564_2022_1105_MOESM4_ESM.xlsx).
+
+First, we `import` python modules needed:
+
+```python
+#import modules;
+import yaml
+import pandas as pd
+import numpy as np
+from shedding_hub import folded_str
+```
+
+Raw data, which is stored on [Shedding Hub](https://github.com/shedding-hub/shedding-hub/tree/main/data/ke2022daily), will be loaded and cleaned to match the most updated [schema](https://github.com/shedding-hub/shedding-hub/blob/main/data/.schema.yaml). The concentration of SARS-CoV-2 RNA in nasal swab samples and saliva samples were calculated based on the calibration curves provided in [Ke et al. (2022)](https://www.nature.com/articles/s41564-022-01105-z).
+
+```python
+#load the data;
+Ke2022 = pd.read_excel('41564_2022_1105_MOESM4_ESM.xlsx',sheet_name='data_samples')
+Ke2022SymptomTime = pd.read_excel('symptom_date_gap.xlsx',sheet_name='Sheet1')
```

Comment:
If `symptom_date_gap.xlsx` is a file we generated from the figure what are your thoughts on storing it as CSV because it's not primary data? Makes it easier to read things on GitHub without having to download.

Reply:
Yes, the validation should pass even if it's not in the schema. The `attributes` are quite flexible: only declared fields are validated, and we can add any extra information. Only the `measurements` are quite strict.

Reply:
Sounds good. Will change it to a CSV file. Thank you.